### PR TITLE
Fix PyTorch XPU Jupyter Documentation

### PR DESCRIPTION
**Pull request by sramakintel**
## Description

Updated documentation to reflect the correct XPU docker image names and tags.

## Related Issue

link to issue https://github.com/intel/ai-containers/issues/207
## Changes Made

- 
- [ ] The code follows the project's [coding standards](../CONTRIBUTING.md#code-style).
- [ ] No Intel Internal IP is present within the changes.
- [x] The documentation has been updated to reflect any changes in functionality.

## Validation

- [ ] I have tested any changes in container groups locally with [`test_runner.py`](../test-runner/README.md) with all existing tests passing, and I have added new tests where applicable.

### CODE REVIEW

File: pytorch/README.md [hunk 1]
```diff
@@ -45,9 +45,10 @@ The images below additionally include [Jupyter Notebook](https://jupyter.org/) s
 
 | Tag(s)                | Pytorch  | IPEX          | Driver | Jupyter Port | Dockerfile      |
 | --------------------- | -------- | ------------- | ------ | ------------ | --------------- |
-| `xpu-jupyter`         | [v2.1.0] | [v2.1.30+xpu] | [803]  | `8888`       | [v0.4.0-Beta]   |
```
Comment (tylertitsworth) [inline pytorch/README.md:48]: What is this tag https://hub.docker.com/layers/intel/intel-extension-for-pytorch/xpu-jupyter/images/sha256-fcf8e51efb2d0e62e01a90052eee8b45b72882be907953a82510b59b82b77cc6?context=explore?

File: pytorch/README.md [hunk 2]
```diff
@@ -45,9 +45,10 @@ The images below additionally include [Jupyter Notebook](https://jupyter.org/) s
...
-| `2.1.20-xpu-pip-base` | [v2.1.0] | [v2.1.20+xpu] | [803]  | `8888`       | [v0.3.4]        |
-| `2.1.10-xpu-pip-base` | [v2.1.0] | [v2.1.10+xpu] | [736]  | `8888`       | [v0.3.4]        |
+| `2.1.20-xpu-pip-jupyter` | [v2.1.0] | [v2.1.20+xpu] | [803]  | `8888`       | [v0.3.4]     |
+| `2.1.20-xpu-idp-jupyter` | [v2.1.0] | [v2.1.20+xpu] | [803]  | `8888`       | [v0.3.4]     |
+| `2.1.10-xpu-pip-jupyter` | [v2.1.0] | [v2.1.10+xpu] | [736]  | `8888`       | [v0.3.4]     |
+| `2.1.10-xpu-idp-jupyter` | [v2.1.0] | [v2.1.10+xpu] | [736]  | `8888`       | [v0.3.4]     |
```
Comment (tylertitsworth) [inline pytorch/README.md:51]: there is a separate section for idp.

**Review by tylertitsworth — CHANGES_REQUESTED** [review]
Please read over the entire document to understand the format.

File: pytorch/README.md (hunk 1 above)
Comment (sramakintel) [reply pytorch/README.md:48]: that is a very old version of the image from 9 months ago and certainly not the latest.

File: pytorch/README.md (hunk 2 above)
Comment (sramakintel) [reply pytorch/README.md]: made changes.

**Review by tylertitsworth — DISMISSED** [review]

**Review by tylertitsworth — APPROVED** [review]

**Pull request merged**